docs/usage/types.md

```diff
@@ -660,3 +660,30 @@ to get validators to parse and validate the input data.
 {!.tmp_examples/custom_data_types.py!}
 ```
 _(This script is complete, it should run "as is")_
+
+## Parsing data into a specified type
+
+Pydantic includes a utility function `parse_as_type` that can be used to apply pydantic's
+parsing logic in an ad-hoc way.
+
+This is especially useful when you want to parse results into a type that is not a direct subclass of `BaseModel`.
+For example: 
+
+```python
```

Comment:
can you make this a file in examples so, then use `print()` below, then print output will automatically be updated.

You'll need to add requests to `docs/requirements.txt` and use a real (very stable) url below. Might be easier to just define a variable `response_data` and add a comment saying it could come from an API call.